test/test_strings.py

```diff
@@ -28,9 +29,11 @@ def _parse_test(self, filename):
         """Helper function to parse a binary file and check whether
         the given string is in the parsed result"""
         self.strings.filename = os.path.join(BINARIES_PATH, filename)
-        binutils_strings = subprocess.check_output(["strings", self.strings.filename])
```

Comment:
Reading from file automatically convert \r\n to \n in Windows. So,  I replaced subprocess.check_output with subprocess.call with output redirection.

Reply:
huh, that works.